chia/cmds/passphrase_funcs.py

```diff
@@ -325,7 +326,7 @@ async def async_update_daemon_migration_completed_if_running() -> None:
         print("Missing root_path in context. Unable to notify daemon")
         return None
 
-    async with acquire_connection_to_daemon(root_path, quiet=True) as daemon:
+    async with acquire_connection_to_daemon(root_path, load_config(root_path, "config.yaml"), quiet=True) as daemon:
```

Comment:
Offhand, it looks a bit odd that the config is loaded in this function but passed into the one in the diff segment above.  Maybe it should be passed in here as well?

Reply:
You mean pass it into `async_update_daemon_migration_completed_if_running`? Why?

Reply:
We discussed this on a call.  While the theory stands, there's too much else around this that needs adjusted.  We'll deal with config loading consistency later.